### PR TITLE
Added FinanceRepo.java file

### PR DESCRIPTION
**Pull request by S4kRiPp4**
Co-Authored by @KyleZim97 @Jesusxx-240 @snowjtSD 

-created FinanceRepo.java file
-has 4 different queries inside

1. query data for current week order count
2. query data for current week dollar sum
3. query data for previous week order count
4. query data for previous week dollar sum

-displays all query variables inside a pop-up using a JOptionPane along with string text using html
-added comments throughout so that this can be used as a reference in creating MarketRepo.java
-weeklyFinance actionEvent inside OrdersGUI.java calls method from FinanceRepo.java
-Math.round was added inside FinanceRepo.java, to round diffDollar variable to hundredths decimal place
*Notes*
-basically this file completes Initiative 3 Epic A,B,C. By displaying query results in a pop-up within our GUI
-Kyle helped a ton w/ the file structure and the SQL statements
-Jesus and Jordan were also both present too and we all worked together over discord via screen sharing
FinanceRepoUX

### CODE REVIEW

**Review by snowjtSD — APPROVED** [review]
GJ

**Review by Jesusxx-240 — APPROVED** [review]
Good job. We spent a lot of time figuring this out, so I'm glad to see it working.
